### PR TITLE
Nested order_by path + unittest

### PR DESCRIPTION
**Pull request by sm0k**
Patch for order_by method on nested field

## Description
`field_object = self.model._meta.fields_map.get(field_name)`
Becomes
`field_object = model._meta.fields_map.get(field_name) if self.model != model else self.model._meta.fields_map.get(field_name)`
in queryset.py

## Motivation and Context
https://github.com/tortoise/tortoise-orm/issues/326

## How Has This Been Tested?
See py.test tests/test_order_by_nested.py

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: tortoise/queryset.py [hunk 1]
```diff
@@ -184,7 +184,10 @@ def resolve_ordering(
                 annotation_info = annotation.resolve(self.model, table)
                 self.query = self.query.orderby(annotation_info["field"], order=ordering[1])
             else:
-                field_object = self.model._meta.fields_map.get(field_name)
+                if self.model != model:
```
Comment (grigi) [inline tortoise/queryset.py]: Would this not mean that when self.model == model the `model` var is still the correct one?
Just do the `field_object = model._meta.fields_map.get(field_name)` instead.

File: tortoise/queryset.py [hunk 2]
```diff
@@ -184,7 +184,10 @@ def resolve_ordering(
...
+                    field_object = model._meta.fields_map.get(field_name)
+                else:
+                    field_object = self.model._meta.fields_map.get(field_name)
                 if not field_object:
                     raise FieldError(f"Unknown field {field_name} for model {self.model.__name__}")
```
Comment (grigi) [inline tortoise/queryset.py]: This should be `{model.__name__}`

**Comment by sm0k** [comment]
So I ran the following commands locally (from Makefile):
```
isort -rc tortoise/ examples/ tests/ conftest.py
black -l 100 --py36 tortoise/ examples/ tests/ conftest.py
```
Wich modified many files (see last commit)
Unit tests are all green on my computer
But Travis jobs are all red now :D 
Sorry im not familiar enough with these processes...

File: tortoise/queryset.py (hunk 1 above)
Comment (sm0k) [reply tortoise/queryset.py]: I tried to just replace self.model with model but didn't seems to work

**Comment by lntuition** [comment]
Ah it's generated by auto styling.. I miss checked it ! sorry.

**Comment by sm0k** [comment]
Looks good now I guess?

**Comment by sm0k** [comment]
> Ah it's generated by auto styling.. I miss checked it ! sorry.
> p.s) @smok found that it is generated by wrong black version which should be fixed

Was in fact due to isort 4.3.4 on my box

File: tortoise/queryset.py (hunk 1 above)
Comment (lntuition) [reply tortoise/queryset.py]: I found that just change model is okay with TCs. How about remove unnecessary logic

``` diff
diff --git a/tortoise/queryset.py b/tortoise/queryset.py
index d09ce89..68f6647 100644
--- a/tortoise/queryset.py
+++ b/tortoise/queryset.py
@@ -184,13 +184,9 @@ class AwaitableQuery(Generic[MODEL]):
                 annotation_info = annotation.resolve(self.model, table)
                 self.query = self.query.orderby(annotation_info["field"], order=ordering[1])
             else:
-                if self.model != model:
-                    field_object = model._meta.fields_map.get(field_name)
-                else:
-                    field_object = self.model._meta.fields_map.get(field_name)
-
+                field_object = model._meta.fields_map.get(field_name)
                 if not field_object:
-                    raise FieldError(f"Unknown field {field_name} for model {self.model.__name__}")
+                    raise FieldError(f"Unknown field {field_name} for model {model.__name__}")
                 field_name = field_object.source_field or field_name
                 field = getattr(table, field_name)

```

**Comment by grigi** [comment]
Thanks for the great work!

**Pull request merged**